pdk/1.0/modules/ROOT/pages/policies-pdk-compile-policies.adoc

```diff
@@ -4,38 +4,33 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images
 
-You can compile the custom policy project after you complete all the steps in xref:policies-pdk-create-project.adoc[].
-
-You must compile the custom policy before you can deploy it to a Flex Gateway.
-
-After you compile your project, test your policy with the following tutorials:
-
-* xref:policies-pdk-debug-local.adoc[]
-* xref:policies-pdk-apply-policies.adoc[]
+Compile your custom policy to create the policies binary target files by using the `make build` command. You must compile your custom policy after every time you edit the `src/lib.rs` file before you can deploy the policy to a Flex Gateway.
```

Comment:
instead of saying "after every time you edit the `src/lib.rs` file". I would say "after every time you change the policy source code". Because the user may add other source code files inside `src/`. Also we could say "after every time you change the content of the `src/` folder".